### PR TITLE
Check metrics

### PR DESCRIPTION
**Pull request by mumichae**
Making sure all metrics are doing what we expect them to.

### CODE REVIEW

File: scIB/metrics/graph_connectivity.py [hunk 1]
```diff
@@ -26,6 +26,6 @@ def graph_connectivity(adata, label_key):
             connection='strong'
         )
         tab = pd.value_counts(labels)
-        clust_res.append(tab[0] / sum(tab))
+        clust_res.append(tab.max() / sum(tab))
```
Comment (LuckyMD) [inline scIB/metrics/graph_connectivity.py:29]: should this be argmax or max?

**Review by LuckyMD — CHANGES_REQUESTED** [review]
Looks good overall!

File: scIB/metrics/graph_connectivity.py (hunk 1 above)
Comment (mumichae) [reply scIB/metrics/graph_connectivity.py:29]: max, if we want the number of nodes in the largest connected component, argmax if we want its label

**Comment by mumichae** [comment]
This PR shouldn't have changed any metrics except for graph connectivity. Here are the scores from the pipeline on test data
image

**Review by LuckyMD — APPROVED** [review]

**Pull request merged**